recipe/conda_build_config.yaml

```diff
@@ -9,22 +9,22 @@ libmlir:
 
 # The package fails to build against CUDA 12.0, upstream uses 12.4.99,
```

Comment:
Please delete everything below this line and rerender

Reply:
Do you want me to add CUDA 11.8 variants then, or should I add `skip` for them in the recipe?

Reply:
I've assumed the latter. Please let me know if I should change it back.

Reply:
We can merge this as-is, but as long as the builds don't make problems (which cannot be said for pytorch, for example), there's no need to drop 11.8

Reply:
Generally, I agree. However, here I've followed the assumption that this will be used along with PyTorch, so the <12 packages are unlikely to ever be installed.